apps/api_web/config/config.exs

```diff
@@ -38,9 +38,10 @@ config :api_web, :versions,
     "2018-07-23",
     "2019-02-12",
     "2019-04-05",
-    "2019-07-01"
+    "2019-07-01",
+    "2020-04-21"
   ],
-  default: "2019-07-01"
+  default: "2020-04-21"
```

Comment:
Can we make the new version `2020-05-01`, and not change the default yet? Since anonymous API usage uses the default version, I'd like to give people a little more time to update.

Reply:
I assume we should also make a followup ticket to bump the default version once 5/1 rolls around?

Reply:
Once this is on production, people can update their applications without waiting for the default to change.

Reply:
Don't we still want to make it the default eventually, though, especially if we want to eventually discourage use of `shapes` for things that are now better done with `route_patterns`?

Reply:
Sorry, that's what the :+1: was for. We'll land this to give people time to update, and then when the date comes, we'll update the default.